docs/core/introduction.md

```diff
@@ -6,169 +6,63 @@ ms.date: 02/24/2022
 ms.custom: "updateeachrelease"
 recommendations: false
 ---
-# What is .NET? Introduction and overview
+# What is .NET?
 
-.NET is a free, open-source development platform for building many kinds of apps, such as:
+.NET is a free, cross-platform, open source developer platform for [building applications](tutorials/with-visual-studio-code.md). .NET is built on a [high-performance runtime](https://devblogs.microsoft.com/dotnet/category/performance/) that is used in production by many [high-scale apps](https://devblogs.microsoft.com/dotnet/category/developer-stories/).
+
+You can [download and use .NET](https://dotnet.microsoft.com/download) for building many types of apps.
+
+**Cloud apps**
 
 * [Web apps, web APIs, and microservices](/aspnet/core/introduction-to-aspnet-core#recommended-learning-path)
 * [Serverless functions in the cloud](/azure/azure-functions/functions-create-first-function-vs-code?pivots=programming-language-csharp)
 * [Cloud native apps](../architecture/cloud-native/index.md)
+
+**Cross-platform client apps**
+
 * [Mobile apps](https://dotnet.microsoft.com/learn/xamarin/hello-world-tutorial/intro)
-* Desktop apps
```

Comment:
I'd add this back and link to https://dotnet.microsoft.com/apps/desktop which is being updated to showcase MAUI and Blazor (PR is being reviewed and should be live shortly). "Desktop" can mean web or native.

Reply:
This was confusing since I moved that content elsewhere. I added the link back.

https://github.com/dotnet/docs/pull/29275/commits/341d96fbf21342f0b1bbeec2c69ddf017b20a1fb

Reply:
My point is desktop development is cross platform so should have a link here. The desktop topic on the dot.net site will skew towards cross platform clients.

Reply:
I understand now.

https://github.com/dotnet/docs/pull/29275/commits/70c9ee2edc7c6855b7d24623b6e21bcf69f89bc6

Reply:
Now that I look at this more closely, I'm confused. IMO, docs should only link to the marketing website for downloads not for information. The marketing website website should link for docs for more information. The back-and-forth suggests we've got the information architecture wrong. That's right isn't it?